README.md

```diff
@@ -10,8 +10,26 @@ Currently this library supports the following job schedulers:
 ### Dependencies
 * [RoddyToolLib](https://github.com/eilslabs/RoddyToolLib)
 
+## Build
+
+Building is as simple as
+
+```bash
+./gradlew build
+```
+
+If you are behind a firewall and need to access the internet via a proxy, you can configure the proxy in `$HOME/.gradle/gradle.properties`:
+
+```groovy
+systemProp.http.proxyHost=HTTP_proxy
```

Comment:
Can it use the environment values?
I normally use the command line options which are more versatile.